balabit/syslog-ng
# DBLD: Support Ubuntu Bionic and update existing images

Updated:
see my last comment: https://github.com/balabit/syslog-ng/pull/2318#issuecomment-427330823
Mainly the PR is about to add Ubuntu Bionic as new supported OS.
The PR is also added some fixes to other parts of the DBLD

Other changes that this PR is introduce:
- added new type of packages file: This file will contains all of the packges for all platforms. There are 
too many package files (apt, pip - for each platform and for all), and it is not so easy to maintain them.
- make target: add support to run make for custom images
- make-check target: add support to run make-check

How to test:
$ dbld/rules image-bionic 
$ dbld/rules bootstrap-bionic 
$ dbld/rules make-bionic 
$ dbld/rules make-check-bionic

Todo:
I also wanted to update old images to the newer form.
Check there is no failures with the new images.

Signed-off-by: Andras Mitzki <andras.mitzki@balabit.com>
